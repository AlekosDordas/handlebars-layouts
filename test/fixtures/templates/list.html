{{#extend "list"}}
	{{#content "list-content" mode="data" as="first-list-items"}}
		<p>First Item</p>
	{{/content}}
	{{#content "list-content" mode="data" as="first-list-items"}}
		<p>Second Item</p>
	{{/content}}
	{{#content "list-content" mode="data" as="first-list-items"}}
		<p>Third Item</p>
	{{/content}}
	{{#content "list-content" mode="data"}}
		<p>This part will go to "block-items" array</p>
	{{/content}}
{{/extend}}
