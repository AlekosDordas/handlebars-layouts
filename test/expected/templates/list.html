<div class="list">
		<h2>First List</h2>
			<div class="list__item">		<p>First Item</p>
</div>
			<div class="list__item">		<p>Second Item</p>
</div>
			<div class="list__item">		<p>Third Item</p>
</div>
		<h2>Second List</h2>
			<div class="list__item">		<p>This part will go to "block-items" array</p>
</div>
</div>
